lemminx-liberty/pom.xml

```diff
@@ -34,6 +34,14 @@
     </distributionManagement>
 
     <build>
+        <resources>
```

Comment:
I think I figured out the problem. When you added this resource section, it caused it to stop including the `src/main/resources` which happens by default. Need to add this section here:

```
            <resource>
                <directory>src/main/resources</directory>
            </resource>
```

Reply:
This fix worked for me locally as well.